src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -2425,4 +2467,49 @@ private class PendingAutoEvoPrediction
 
         public bool Finished => AutoEvoRun.Finished;
     }
+
+    /// <summary>
+    ///   Extension of LineChart's default datasets dropdown legend to allow sectioning of extinct species.
+    /// </summary>
+    private class SpeciesPopDatasetsLegend : LineChart.DataSetsDropdownLegend
```

Comment:
I'd put this in a separate file to try to keep down the length of this file, as this has now reached 2500 lines.